charts/openstack-cluster/values.yaml

```diff
@@ -93,6 +93,12 @@ registryMirrors:
   registry.k8s.io:
     - https://quay.io/v2/azimuth/registry.k8s.io
 
+# Proxy configuration for containerd.
+proxy:
+  http_proxy:
+  https_proxy:
+  no_proxy:
```

Comment:
These should be `camelCased` to match Helm values conventions, e.g. `httpProxy`, `httpsProxy`, `noProxy`.